pkg/procmanager/procmanager.go

```diff
@@ -33,7 +34,7 @@ func NewProcManager(dryRun bool) *ProcManager {
 }
 
 // RunRepeatWithRetries performs a Run multiple times with retries.
-// If an error occurs during the repetition, the last error will be returned.
+// If any error occurs during the repetition, all errors will be aggregated and returned.
```

Comment:
So is this comment incorrect and we always returned all errors?

Reply:
yes, `aggErrors = append(aggErrors, innerErr)`